### ISSUE TITLE
[$20][MSFT-241] [Register][NVD][Edge] Social media icons are not identified correctly sometimes when hovering by the screen-reader

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2896]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member/registration
2. Hover over the social media icons, don't hover over the social media icon text

**Expected Result**
Should read out the name of the social media when hovering over the image

**Actual Result**
Social media icons are not identified correctly sometimes when hovering by the screen-reader

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Edge

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=false&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F75

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096235 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096235 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096235 has been updated - it has been assigned to nahidshahin.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by nahidshahin** [comment]
PR https://github.com/appirio-tech/accounts-app/pull/233

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2896 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096741 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096741 has been updated - it has been assigned to nahidshahin.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by mishacucicea** [comment]
@nahidshahin This doesn't fix the mouse-over issue. Can you try to find another solution?

**Comment by nahidshahin** [comment]
@mishacucicea I don't found any issue regarding mouse-over at description. What is the issue here?

**Comment by mishacucicea** [comment]
@nahidshahin The issue is that if you use a screen-reader (e.g. https://www.nvaccess.org/download/), it won't readout the title (https://www.mediacurrent.com/blog/dont-rely-title-attribute-accessibility-seo/).

@drasticdpk / @tosha5252  Not sure if this would make much sense in the context of a screen-reader (e.g. a visually impaired person won't hover over items). We can have a fix to show a title/tooltip on the items, but the screen-reader won't action on hover, only when using tab.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096741 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
Already fixed by PR https://github.com/appirio-tech/accounts-app/pull/244 
@veshu ticket prize yet to be doubled. Thanks!

**Comment by veshu** [comment]
@mishacucicea please check the prize part. Thanks!

**Comment by mishacucicea** [comment]
@r0hit-gupta I don't understand what you're saying? Contact me on slack.

**Comment by r0hit-gupta** [comment]
@mishacucicea can you please provide me with the slack details? I do not have any information about the Slack channel. Thanks! 
Also, the fix is working for me on hover too. Please confirm on https://accounts.topcoder-dev.com/member/registration. If there is still an issue, please let me know the screen reader being used so I can test using the same reader. Thanks!

**Comment by lakshmiathreya** [comment]
Working as expected in Win/Chrome/NVDA. Putting on hold since this is and Edge issue. Can be reviewed later.

**Comment by crazyk07** [comment]
Just tested this is Prod with Edge and NVDA.  Works perfectly.  May have been solved by another issue. 
 Ignore reference above.  Closing issue.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096741<br/><br/>```This is an automated message for crazyk via Topcoder X```
